### DISCUSSION TITLE
Changing Margins

### DISCUSSION THREAD

**Discussion by anshjhatakia** [jakewies/hugo-theme-codex#151] [Q&A] [answered]
Hey everyone!

Does anyone have an idea on how to change margins on blog posts? When I'm using my desktop or laptop, I'd like to have text take up a bit more of the screen. Same with code blocks and such. I'm just not sure where to start.

Thanks for the help!

**Comment by jakewies** [comment, marked as answer]
Hey @anshjhatakia 👋  You're the first to use a discussion in this project, so welcome! We have a documented way of adding custom styling to the site [in the README](https://github.com/jakewies/hugo-theme-codex#custom-styling). You can give this a go and see if it works for you. It's been a while since I've worked on this project (real life has gotten in the way), but many users of this theme have had success with this technique. 

Let me know if this works out or not! Happy to help you find the right solution.

EDIT: If you go with the `custom.scss` option (which is really your only option at this point), you should be able to override this class's style:

![Screen Shot 2021-01-19 at 10 44 13 AM](https://user-images.githubusercontent.com/12075916/105057580-5b89d380-5a43-11eb-8a21-b906508341cd.png)

With your own:

```css
/* assets/scss/custom.scss */

.post {
  max-width: 100%; /* or whatever you want */
}
```

`custom.scss` is rendered after the page styles, so styles defined here will be given priority.

**Reply by anshjhatakia** [reply]
This worked flawlessly! Thank you so much Jake!

**Reply by jakewies** [reply]
Awesome! No worries at all. Enjoy the theme 😄
